src/terraform/commands.go

```diff
@@ -3,14 +3,30 @@ package terraform
 import (
 	"fmt"
 
+	"github.com/Originate/exosphere/src/docker/tools"
 	"github.com/Originate/exosphere/src/types"
 	"github.com/Originate/exosphere/src/util"
 )
 
 // RunInit runs the 'terraform init' command and force copies the remote state
 func RunInit(deployConfig types.DeployConfig) error {
+	homeDir, err := util.GetHomeDirectory()
+	if err != nil {
+		return err
+	}
+	volumes := []string{
+		fmt.Sprintf("%s:/app", deployConfig.TerraformDir),
+		fmt.Sprintf("%s/.aws:/root/.aws", homeDir),
+	}
 	backendConfig := fmt.Sprintf("-backend-config=profile=%s", deployConfig.AwsConfig.Profile)
-	return util.RunAndPipe(deployConfig.TerraformDir, []string{}, deployConfig.Writer, "terraform", "init", "-force-copy", backendConfig)
+	dockerRunConfig := tools.RunConfig{
```

Comment:
This could also be defined inline as the parameter instead of creating a variable. Same for below